charts/pvc-autoresizer/templates/controller/clusterrole.yaml

```diff
@@ -1,34 +1,42 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "pvc-autoresizer.fullname" . }}-controller
+  name: {{ template "pvc-autoresizer.fullname" . }}-controller-storageclasses
   labels:
     {{- include "pvc-autoresizer.labels" . | nindent 4 }}
 rules:
 - apiGroups:
-  - ""
+  - storage.k8s.io
   resources:
-  - events
+  - storageclasses
   verbs:
-  - create
   - get
   - list
   - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "pvc-autoresizer.fullname" . }}-controller-persistentvolumeclaims
```

Comment:
This ClusterRole contains a rule for both PVC and events, so the suffix `-controller-persistentvolumeclaims` seems not good.
How about just removing it?
```
name: {{ template "pvc-autoresizer.fullname" . }}-controller
```